R/revenuedata.R

```diff
@@ -76,16 +76,22 @@ RevenueData <- function(value, from, to, start = min(from), end = max(from), id,
                         profiling = NULL, trim.id = 50) #, tolerance = .01)
 {
     # Checking the input variables.
-    n = length(value)
+    n <- length(value)
     checkVariableForLengthAndMissingData(value, n)
     checkVariableForLengthAndMissingData(from, n)
     checkVariableForLengthAndMissingData(to, n)
     checkVariableForLengthAndMissingData(id, n)
 
+    all.args.valid <- vapply(list(from, to, start, end), inherits, logical(1L), c("Date", "POSIXlt", "POSIXct"))
```

Comment:
Wasn't there an error if e.g. one was Date and the other POSIXct?

Reply:
check_tzones errored with the old lubridate and warned with the new. This error was more inline to align with what is expected input args.